operator/Makefile

```diff
@@ -65,11 +66,21 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build -t ${IMG}:latest .
+	docker tag ${IMG}:latest ${IMG}:${COMMIT_HASH}
+	docker tag ${IMG}:latest ${IMG}:dev
 
 docker-push: ## Push docker image with the manager.
-	docker tag ${IMG} localhost:37129/${IMG}
-	docker push localhost:37129/${IMG}
+	docker push ${IMG}:latest
```

Comment:
we do not need a docker push command, we can add it to our ci scripts and if users are pushing their own image this will not work anyway